server/s3_global_bucket_index_metadata.cc

```diff
@@ -29,7 +29,8 @@ extern struct m0_uint128 global_bucket_list_index_oid;
 extern struct m0_uint128 replica_global_bucket_list_index_oid;
 
 S3GlobalBucketIndexMetadata::S3GlobalBucketIndexMetadata(
-    std::shared_ptr<S3RequestObject> req, std::shared_ptr<MotrAPI> motr_api,
+    std::shared_ptr<S3RequestObject> req, std::string bucket,
```

Comment:
Same comment here. What is the reason to add new parameter (i.e. bucket name) to constructor?

Reply:
fixed this now.